### ISSUE TITLE
No holding <SHIFT>, but craft x10 (sometimes)

### ISSUE THREAD

**Issue by leonhetch** [thmsndk/Raft-BulkCrafting#2]
https://user-images.githubusercontent.com/26923992/133742679-2edf39fe-6f35-4b6c-98e8-4179f3c535b4.mp4

**Comment by thmsndk** [comment]
Thanks for the report @leonhetch could you describe in more detail what you are doing on your PC? what you are clicking? Are you alt tabbing? It looks like you hare holding shift down when you open the interface, and looks you keep holding it down while swapping recipes.

I would very much appreciate a step by step description of what you are doing. I can see what happens in the video, but I am unsure on what you are doing that triggers that behavior.

**Comment by thmsndk** [comment]
Are you still experiencing this issue with the latest release?

**Comment by DefectiveNu** [comment]
hi, I am not the original reporter but I've gotten this issue myself.  However, it wasn't crafting 10x, but some odd number.  Sometimes when I crafted collection nets, I would get 3 and see the item notification for +1, +2.  It also doesn't take the resources for the "bonus" items

![20220729235320_1](https://user-images.githubusercontent.com/27719173/182105436-7967dec1-c804-46cd-bf51-c76dce119b91.jpg)

**Comment by thmsndk** [comment]
Yeah, I haven't found a reliable reproduce for crafting odd numbers, I assume you are also using Craft From All Storage?
It is really strange, because bulk crafting should be multiplying it with 10 or not at all.
I seem to recall a potential issue if you have modified stacksizes with the stacksize mod, have you done that?
